documentation/book/proc-using-openshift-builds-create-image.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-using-kafka-connect-with-plugins.adoc
+
+include::common/attributes.adoc[]
+
+[id='using-openshift-s2i-create-image-{context}']
+= Using {OpenShiftName} builds and S2I to create new images
+
+{OpenShiftName} supports link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^] which can be used together with the link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new Docker images.
```

Comment:
Replace _Docker images_ with _container images_. The same is also in the next sentences. (the reason: @ncbaratta mentioned some requirements to avoid mentioning Docker ... that makes sense especially for downstream).